src/static/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Queues@Illinois",
+  "name": "Queues@Dev",
```

Comment:
This is the the same manifest deployed to production, so this should stay as it was.